### DISCUSSION TITLE
Monthly ESMValTool community meeting June 2023

### DISCUSSION THREAD

**Discussion by bouweandela** [ESMValGroup/Community#105] [Meetings]
Dear @ESMValGroup/esmvaltool-developmentteam,

The next online community meeting will be on

Tuesday, June 20, at 3PM CEST

The duration will be one hour. [Join the zoom meeting](https://us02web.zoom.us/j/86356385817).

Calendar invitations to the meeting are sent to the mailing list, join it [here](https://docs.esmvaltool.org/en/latest/introduction.html#user-mailing-list).

**Format and agenda**

- Anyone with an interest in ESMValTool is welcome to join and ask questions
- We start the meeting by doing a round where everyone gets one minute to introduce themselves and tell what they will be working on in the next month or what they would like help with.
- Using the input from the introduction round, we choose some topics to talk about in more detail.
- If you already know that you would like to bring up a particular topic in the meeting, you can mention the [GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues) on the topic in a comment below. That will make it easy to find for the other people in the meeting and we can already start discussing there before the meeting. Just [create a new GitHub issue](https://github.com/ESMValGroup/ESMValTool/issues/new) about your topic if there is no open issue yet.
- GitHub issues are our preferred way to communicate outside the meeting and keep track of work, so any feature requests/action items/etc discussed in the meeting should ideally result in a GitHub issue.

Notes from the previous meeting are available in #97

**Comment by bouweandela** [comment]
This will be the last community meeting that I organize. If you've enjoyed these meetings for the past few years and would like them to continue: the @ESMValGroup/userengagementteam is looking for someone who will organize them in the future, so please get in touch. The time investment is about 3 hours a month: 1 hour to prepare the meeting, 1 hour to host it, and 1 hour to write up the meeting notes.

**Comment by bsolino** [comment]
I would like to present the updates I've done to the ESMValTool HTML output. It is still in development, but it is already fully functional.
Furthermore, I have some questions about that project that I wanted to bring to the community. Detailed information (and the questions) are in the first post of issue https://github.com/ESMValGroup/ESMValCore/issues/2059

**Comment by glpotter** [comment]
I would like to ask if there is any interest in issue #101

**Comment by bouweandela** [comment]
Updates

- @remi-kazeroni Announced DKRZ help with making preprocessor functions lazy
- @bsolino Improving HTML output and updating CVDP
- @LisaBock Handing over @ESMValGroup/userengagementteam leadership to Tammy Palmer, working on CMORizing ESA CCI cloud data
- @catherinehardacre Working on areasol optical depth diagnostic, planned for v2.10
- @katjaweigel Planning to look into ocean variables
- @zklaus Preparing next week's workshop, working on aeronet data with @catherinehardacre for ESM2025. Working on fire weather index. Lots of work on fixing packages on conda-forge.
- @glpotter Working on CMORizers for reanalysis data. Would like feedback on https://github.com/ESMValGroup/Community/issues/101. @remi-kazeroni: There is a session planned at next week's workshop. Current status: https://github.com/ESMValGroup/ESMValTool/issues/2851.
- @bouweandela Release manager for upcoming v2.9 release https://github.com/ESMValGroup/ESMValTool/discussions/3198 and helping @Peter9192 with creating two blog posts for promotional purposes

**Reply by valeriupredoi** [reply]
and me, boss? :grin:

**Reply by bouweandela** [reply]
Sorry, my zoom disconnected when you were giving an update. Could you add your a note with your update yourself, please?

**Reply by valeriupredoi** [reply]
sure, my update:
![mechanic-hammer](https://github.com/ESMValGroup/Community/assets/28983971/8982b2fe-9ed3-431a-be4e-907bb9d9313a)

**Comment by bouweandela** [comment]
There was a question on how to view the HTML output when working on an HPC system. Suggestions included:
- Mount the remote filesystem using the file browser provided by your OS ([Ubuntu Linux example](https://dev.to/yousufbasir/add-remote-server-location-to-ubuntu-nautilus-using-ssh-2i77))
- Use VSCode with the [Remote Explorer](https://marketplace.visualstudio.com/items?itemName=ms-vscode.remote-explorer) extension
